### PR TITLE
Updated Django Container Engine sample to use CloudSQL v2, sidecar co…

### PR DESCRIPTION
**Pull request by ryanmats**
…ntainer pod pattern, Django 1.10, and Deployment rather than ReplicationController

Tested this a number of times to make sure it works. Staged changes to the cloud.google.com tutorial at https://cloud-dot-devsite.googleplex.com/python/django/container-engine and will send that changelist for review once this code is merged.

### CODE REVIEW

File: container_engine/django_tutorial/polls.yaml [hunk 1]
```diff
@@ -39,14 +39,41 @@ spec:
...
+            mountPath: /secrets/cloudsql
+            readOnly: true
+          - name: ssl-certs
+            mountPath: /etc/ssl/certs
+          - name: cloudsql
+            mountPath: /cloudsql
+      # [END proxy_container] 
+      # [START volumes]
+      volumes:
+        - name: cloudsql-oauth-credentials
```
Comment (waprin) [inline container_engine/django_tutorial/polls.yaml]: where are these secrets created?

File: container_engine/django_tutorial/README.md [hunk 2]
```diff
@@ -5,187 +5,10 @@ app on Google Container Engine. It uses the [Writing your first Django app](http
 .9/intro/tutorial01/) Polls application as the example app to deploy. From here on out, we refer to this app as
 the 'polls' application.
 
-## Pre-requisites
 
-1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com).
+# Tutorial
+See our [Django on Container Engine](https://cloud.google.com/python/django/container-engine) tutorial for instructions for setting up and deploying this sample application.
```
Comment (waprin) [inline container_engine/django_tutorial/README.md]: There's secrets in here but the tutorial I see now doesn't have secret anywhere?

File: container_engine/django_tutorial/polls.yaml (hunk 1 above)
Comment (ryanmats) [inline container_engine/django_tutorial/polls.yaml]: bill and i just discussed this, will update on Tuesday

File: container_engine/django_tutorial/README.md (hunk 2 above)
Comment (ryanmats) [inline container_engine/django_tutorial/README.md]: bill and i just discussed this, will update on Tuesday

**Pull request merged**